### ISSUE TITLE
Loader Issue

### ISSUE THREAD

**Issue by austin-InDro** [foxglove/create-foxglove-extension#80]
**Description**
Unable to build a custom extension using ReactPlayer from https://github.com/vdalex25/rtsp-to-web-player/blob/main/examples/ReactComponent/src/App.js

I thought the error may be related to #31, but it seems that was merged, so it may be a new issue (Or I'm doing something completely wrong).

- Version: "0.6.1",
- Platform: Ubuntu 22.04 X86

**Steps To Reproduce**

```bash
git clone git@github.com:austin-InDro/camera_streamer.git
npm install
npm run package
```
**Expected Behavior**
`Build successful`

**Actual Behavior**
```bash
> camera_streamer@0.0.1 package
> foxglove-extension package

Executing prepublish script 'yarn run foxglove:prepublish'...
yarn run v1.22.19
$ foxglove-extension build --mode production
Building...
Error: build failed: Errors:
ModuleParseError: Module parse failed: Unexpected token (16:12)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|     }
| 
>     return (<div className="input-group mb-3">
|             <input type="text"
|                    className="form-control"
ModuleParseError: Module parse failed: Unexpected token (40:12)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|     }, [url, player]);
| 
>     return (<div className="player-wrapper">
|         <div ref={playerElement}/>
|         <div className="control">
    at /home/austin/Documents/web/camera_streamer/node_modules/create-foxglove-extension/dist/build.js:63:35
    at /home/austin/Documents/web/camera_streamer/node_modules/webpack/lib/HookWebpackError.js:68:3
    at Hook.eval [as callAsync] (eval at create (/home/austin/Documents/web/camera_streamer/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:6:1)
    at Hook.CALL_ASYNC_DELEGATE [as _callAsync] (/home/austin/Documents/web/camera_streamer/node_modules/tapable/lib/Hook.js:18:14)
    at Cache.shutdown (/home/austin/Documents/web/camera_streamer/node_modules/webpack/lib/Cache.js:150:23)
    at /home/austin/Documents/web/camera_streamer/node_modules/webpack/lib/Compiler.js:1225:15
    at Hook.eval [as callAsync] (eval at create (/home/austin/Documents/web/camera_streamer/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:6:1)
    at Hook.CALL_ASYNC_DELEGATE [as _callAsync] (/home/austin/Documents/web/camera_streamer/node_modules/tapable/lib/Hook.js:18:14)
    at Compiler.close (/home/austin/Documents/web/camera_streamer/node_modules/webpack/lib/Compiler.js:1218:23)
    at /home/austin/Documents/web/camera_streamer/node_modules/create-foxglove-extension/dist/build.js:55:22
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
yarn failed with exit code 1
```

**Comment by defunctzombie** [comment]
I took a quick look at your code and I see that there is a file called `input-group.js` which has JSX syntax. JSX syntax needs to be in `.jsx` or `.tsx` files. I also looked at what we require for our packaging and it seems we are even more strict than that and require use of `.tsx` files instead of `.jsx` files. I recommend renaming any files with JSX syntax to `.tsx` until we add support to build `.jsx` files.
